MoTrPAC/exercise-transcriptome-meta-analysis
# Connects to #13. Connects to #14. Published paper citation and Google Analytics

### Key Changes

1. Implemented UI to reference the published paper citation on both **Home** and **About** pages.
2. Implemented **Google Analytics** on both **Home** and **About** pages.

### Technical Notes

1. Reviewing this PR requires `node.js v14.15.4`.
2. Start the site by running `yarn start` after running `yarn install`.

### Steps to test

1. On the landing page, expect to see a banner UI showing reference to citation.
2. Navigate to the **About** page, expect to see the same banner UI showing reference to citation.
3. Sign into **Google Analytics**, and select the **extrameta** account and then the **development** property.
4. Select the **Realtime** option from the left navbar, and expect to see a blue dot on the map indicating your current geographical location.
